tests/test_activate.sh

```diff
@@ -10,7 +10,16 @@ rm -rf ${TESTENV}
 
 echo "$0: Creating virtualenv ${TESTENV}..." 1>&2
 
-${VIRTUALENV} ${TESTENV} | tee ${ROOT}/tests/test_activate_output.actual
+PY_BLEND=$(python -c "import sys; print('pypy' if '__pypy__' in sys.builtin_module_names else 'python');")
+cat >${ROOT}/tests/test_activate_output.expected <<EOL
```

Comment:
You could use [platform.python_implementation()](https://docs.python.org/2/library/platform.html#platform.python_implementation) instead of probing `sys.builtin_module_names`